### PR TITLE
Adds table of top 5 articles by retention rate

### PR DESCRIPTION
**Pull request by katsuraku**

### CODE REVIEW

File: src/server/queries/topArticleQuery.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ export default function TopArticleQuery(query){
                 view_timestamp: {
                   "from": query.dateFrom,
                   "to": query.dateTo
-                  /*"from": "2016-01-06T00:00:00.000Z",
-                  "to": "2016-01-06T23:59:00.000Z"*/
+                  // "from": "2016-01-06T00:00:00.000Z",
+                  // "to": "2016-01-06T23:59:00.000Z"
```
Comment (radiodario) [inline src/server/queries/topArticleQuery.js]: should we just delete this instead?

**Comment by katsuraku** [comment]
sure. it was just to make it easier to test when we had no data for a few days but it shouldn't stay in the code.

File: src/server/aggregations/TopArticles.js [hunk 2]
```diff
@@ -110,6 +110,33 @@ export default function TopArticlesAggregation(query) {
          }
        }
      }
+   },
+   "top_kept_on_FT" : {
```
Comment (StuartFT) [inline src/server/aggregations/TopArticles.js]: found this var name a bit confusing maybe could be renamed to something closer to output e.g. top_articles_retention?

Comment (katsuraku) [inline src/server/aggregations/TopArticles.js]: :+1:

**Pull request merged**